exaudfclient/base/exaudfclient.template.sh

```diff
@@ -2,6 +2,6 @@
 
 SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 echo "Changing to script directory  $SCRIPT_DIR"
-cd $SCRIPT_DIR
+cd "$SCRIPT_DIR" || return 1
```

Comment:
Reason:
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Reply:
ok, this one, I don't understand

Reply:
Here is some further info:
https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC2164